magento/magento2
# Adding a boolean to erase the deploted version number

### Description (*)

The issue says only to deploy statics, but 

### Fixed Issues (if relevant)

1. magento/magento2#5927: LESS doesn't compile to CSS in production mode

### Manual testing scenarios (*)

1. check the deployed version in pub/static/deployed_version.txt
2. Run the full process of production deploy it will recreate the static files

1. Run the command php bin/magento setup:static-content:deploy -f 
2. Run the command php bin/magento setup:static-content:deploy -f --delete-previous-files

### Questions or comments


### Contribution checklist (*)
